packages/modules/pricing/src/migrations/.snapshot-medusa-pricing.json

```diff
@@ -599,41 +631,31 @@
           "mappedType": "datetime"
         }
       },
-      "name": "price_set_rule_type",
+      "name": "price_rule",
       "schema": "public",
       "indexes": [
         {
-          "keyName": "IDX_price_set_rule_type_price_set_id",
-          "columnNames": [
-            "price_set_id"
-          ],
-          "composite": false,
-          "primary": false,
-          "unique": false,
-          "expression": "CREATE INDEX IF NOT EXISTS \"IDX_price_set_rule_type_price_set_id\" ON \"price_set_rule_type\" (price_set_id) WHERE deleted_at IS NULL"
-        },
-        {
-          "keyName": "IDX_price_set_rule_type_rule_type_id",
+          "keyName": "IDX_price_rule_price_id_rule_type_id_unique",
           "columnNames": [
-            "rule_type_id"
+            "price_id"
           ],
           "composite": false,
           "primary": false,
           "unique": false,
-          "expression": "CREATE INDEX IF NOT EXISTS \"IDX_price_set_rule_type_rule_type_id\" ON \"price_set_rule_type\" (rule_type_id) WHERE deleted_at IS NULL"
+          "expression": "CREATE UNIQUE INDEX IF NOT EXISTS \"IDX_price_rule_price_id_rule_type_id_unique\" ON \"price_rule\" (price_id, rule_type_id) WHERE deleted_at IS NULL"
```

Comment:
todo: This index seems to be wrong as it still has the rule_type_id

Reply:
Yup, solved by removing the type id from below